content/zh/docs/concepts/observability-primer.md

```diff
@@ -39,7 +39,7 @@ cSpell:ignore: webshop
 
 ### 日志
 
-**日志** 日志是由服务或其他组件发出的带时间戳的消息。与[追踪](#distributed-traces)不同，它们不一定与特定的用户请求或事务相关联。在软件中几乎到处都能找到日志。长期以来，开发人员和运维人员一直依靠日志来洞察系统行为。
```

Comment:
Here and elsewhere, you should preserve the original English links, otherwise the fallback page feature might break.

Reply:
Thanks, I have mark it back. But sometimes I edit the file in my local env, I click the link it will jump to an error place, then I thought this might be a problem, but in fact it could works on the website. So how can I check if every link works well from my local env?

Reply:
And as you see, if I put this to original English link, the CI check will fail